doc/sphinx/index.rst

```diff
@@ -48,9 +49,23 @@ Contents:
 
    install
    quickstart
+
+.. toctree::
```

Comment:
Out of curiosity, what's the rationale for having multiple toctrees here instead of a single one?